### PR TITLE
[SMD-232] convert front end validation error message display to table

### PR DESCRIPTION
**Pull request by Connor-OS**
This PR switches the current Accordian based error presentation to one based on a table layout. I've added an ErrorTable macro that can present the API responce in table format and switched around thelogic based on the new format of the JSON

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### Screensho
Screenshot 2023-10-17 at 10-08-39 Submit monitoring and evaluation data – GOV UK
ts of UI changes (if applicable)

### CODE REVIEW

File: tests/test_routes.py
Comment (cyrusdobbs) [inline tests/test_routes.py:29]: We currently don't return this upon success

**Review by cyrusdobbs — COMMENTED** [review]
Looks good 👍 just one comment

File: app/templates/main/errorTable.html
Comment (gidsg) [inline app/templates/main/errorTable.html:2]: Can we use the Jinja macro for this instead?

Comment (Connor-OS) [reply app/templates/main/errorTable.html]: Looks good, I've switched to this way of doing it now

**Review by gidsg — APPROVED** [review]
LGTM (once conflicts are resolved)

**Comment by cyrusdobbs** [comment]
We need to hold on merging this until these backend changes are merged - https://github.com/communitiesuk/funding-service-design-post-award-data-store/pull/295

**Pull request merged**
